metapool/sample_sheet.py

```diff
@@ -537,6 +528,12 @@ def get_lane_number(self):
 
         return int(lanes[0])
 
+    def _get_expected_columns(self, table=None):
```

Comment:
Also this one? if part of the object why `_` as prefix?

Reply:
_get_expected_columns() is meant to be a private method and only used by KLSampleSheet and its children. It's not something users have ever needed, and it's behavior is particular. Users should lean towards using make_sample_sheet() and load_sample_sheet() as their methods to create SampleSheet() objects and hence this method shouldn't have value to the user.